docs/gcp_service_directory/concepts.md

```diff
@@ -0,0 +1,14 @@
+# Concepts
+
+## Google Service Directory
+
+Service directory is a product made by Google and is one example of a service registry. As CN-WAN Operator, it allows for objects to categorized in *Namespace*s, *Service*s and *Endpoint*s.
+
+For more information, visit Service Directory's [official website](https://cloud.google.com/service-directory).
+
+## Usage with CN-WAN Operator
+
+To learn how you can use the CN-WAN Operator with Service Directory, you can follow one of these guides:
+
+* A [quickstart](./quickstart.md) for Dervice Directory and CN-WAN Operator
+* A [guide] on how to configure CN-WAN Operator to correctly connect to Service Directory
```

Comment:
Should guide be a link somewhere? Else remove the []

Reply:
Yeah I forgot to add it